lc-hax/Scripts/Modules/Possession/PossessionMod.cs

```diff
@@ -154,7 +154,7 @@ internal sealed class PossessionMod : MonoBehaviour {
             this.InitCharacterMovement(enemy);
             this.UpdateComponentsOnCurrentState(true);
             this.SetAIControl(false);
-            this.MainEntrance = RoundManager.FindMainEntranceScript(false);
+            this.MainEntrance = FindObjectsOfType<EntranceTeleport>().First(entrance => entrance.entranceId == 0);

```

Comment:
Why?

Reply:
Because the method you used is not reliable, we need the main entrance from the inside, that method just gives a random entrance and even with both true or false, it still gave the wrong one

Reply:
That's not right, `/enter` and `/exit` uses it perfectly fine.

Reply:
> That's not right, `/enter` and `/exit` uses it perfectly fine.

yeah i tried, but somehow the enemy when noclipped still had that jittery and lost movement, this way still works.

Reply:
what?

Reply:
Reverted, just needed to be the right value.